### PR TITLE
Feat/multilevel

### PR DESCRIPTION
**Pull request by matiu**
- adds config.js configuration file
- adds multilevel support.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      url: 'https://test-insight.bitpay.com:443',
+    },
+  },
+  WalletService: {
+    storageOpts: {
+      dbPath: './db',
+      /*  To use multilevel, uncomment this:
+      multiLevel: {
+        host: 'localhost',
+        port: 3002,
```
Comment (isocolsky) [inline config.js]: This port configuration is not used when starting the multilevel process

File: lib/blockchainexplorer.js [hunk 2]
```diff
@@ -14,19 +14,13 @@ function BlockChainExplorer(opts) {
   $.checkArgument(opts);
   var provider = opts.provider || 'insight';
   var network = opts.network || 'livenet';
+  var dfltUrl = network == 'livenet' ? 'https://insight.bitpay.com:443' :
```
Comment (isocolsky) [inline lib/blockchainexplorer.js]: Shouldn't this be on the configuration file?

**Comment by isocolsky** [comment]
This is a huge improvement! Great job!!

File: config.js (hunk 1 above)
Comment (matiu) [inline config.js]: This configuration is meant for the client of the multilevel server, not for the server itself. The `multilevel/` was only included as a demo.

File: lib/blockchainexplorer.js (hunk 2 above)
Comment (matiu) [inline lib/blockchainexplorer.js]: It is on the conf file, I just wanted to give a nice dflt value if the keys are not present of the config file.

**Pull request merged**